src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -413,7 +413,15 @@ public virtual HtmlString GenerateAntiforgery([NotNull] ViewContext viewContext)
             var fullName = GetFullHtmlFieldName(viewContext, expression);
             if (string.IsNullOrEmpty(fullName))
             {
-                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(expression));
+                throw new ArgumentException(
+                    string.Format(
+                        Resources.HtmlGenerator_FieldNameCannotBeNullOrEmpty,
```

Comment:
Use `Resources.FormatHtmlGenerator_FieldNameCannotBeNullOrEmpty(...)` and get rid of the explicit `string.Format()` calls.